src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -375,6 +383,11 @@
   <Target Name="CollectSDKReferencesDesignTime" />
   <Target Name="CollectResolvedSDKReferencesDesignTime" />
 
+  <Target
+    Name="ResolveFrameworkReferencesDesignTime"
+    Returns="@(FrameworkReference)"
+    DependsOnTargets="ResolveFrameworkReferences" />
```

Comment:
Once https://github.com/microsoft/msbuild/pull/4471 and https://github.com/dotnet/sdk/pull/3355 are merged, this will be changed to return `@(ResolvedFrameworkReference)`. Should it be moved somewhere else though?

Reply:
This target isn't needed as you aren't doing anything additional.

Reply:
Isn't it needed until we have the actual target in MSBuild? In this case we're synthesizing the `ResolvedFrameworkReference`.

Once the `ResolveFrameworkReferences` task returns `ResolvedFrameworkReference` then won't this target become:

```xml
<Target name="ResolveFrameworkReferencesDesignTime" />
```